arrows/core/tests/test_derive_metadata.cxx

```diff
@@ -55,6 +55,17 @@ make_metadata()
   return { m1, m2 };
 }
 
+// ----------------------------------------------------------------------------
+kv::image_container_scptr
+make_image()
+{
+  constexpr auto frame_height = size_t{ 720 };
+  constexpr auto frame_width = size_t{ 1080 };
```

Comment:
Not important to me, but curious if there was a reason for not choosing the following?
```suggestion
  constexpr size_t frame_width{ 1080 };
```
And likewise above.

Reply:
Mostly, I am not a fan of `type name{ value }` declarations. In some cases they're the only option, but I feel like `auto name = value // or type{ value }` reads better. (Ultimately, I think the difference can be viewed as "imperative" vs. "declarative".)